python/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+from setuptools.dist import Distribution
+
+class BinaryDistribution(Distribution):
+    """Distribution which always forces a binary package with platform name
+    """
+    def has_ext_modules(self):
+        return True
+
+path_to_file = "../pointmatcher/PointMatcher.h"
```

Comment:
(!) Hardcoding path break python portability (to non unix OS in that case). Use [`os.path` package](https://docs.python.org/3/library/os.path.html) to define path instead, it will handle OS specific filesystem difference.
``` suggestion
path_to_file = os.path.relpath(os.path.join("pointmatcher", "PointMatcher.h"))
```